packages/react/src/components/logged-in/logged-in.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { LoggedIn } from ".";
+
+const meta: Meta<typeof LoggedIn> = {
+  component: LoggedIn,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof LoggedIn>;
+
+export const Primary: Story = {
+  render: () => (
+    <LoggedIn withFactorMethods={["email_link", "otp_via_sms"]}>
+      Only visible for the users authenticated with email magic link and SMS
+      code.
+    </LoggedIn>
+  ),
+};
+
+export const Secondary: Story = {
```

Comment:
I think that primary/secondary made sense for the button example because it was showing the primary and secondary buttons, but here it feels like without opening these stories I wouldn't know what they contain.

I'm not sure how to define custom story names with this object pattern but when defining a story using function you can do it like this:
```tsx
const Secondary = () => <LoggedIn withFactoryMethods={...} />
Secondary.storyName = "With factor count check"
```

Do you think we can improve discoverability of stories like this?

Reply:
yes I agree, thanks for the suggestion 👍